src/main.ts

```
@@ -1,26 +1,70 @@
+  if (!messageArea) return;
+  messageArea.textContent = message;
+    messageArea.classList.remove('hidden', 'bg-red-100', 'text-red-700', 'bg-green-100', 'text-green-700', 'bg-blue-100', 'text-blue-700');
+
+    switch (type) {
+      case 'error':
+        messageArea.classList.add('bg-red-100', 'text-red-700');
+        break;
+      case 'success':
+        messageArea.classList.add('bg-green-100', 'text-green-700');
+        break;
+      case 'info':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)